### PR TITLE
Fix #16201 - Trim spaces for integer values in table search

### LINKED ISSUE #16201: Please enter a valid number.
"2710 " (note the space) isn't considered a valid number, by the client-side validator. I think the server-side code does consider this valid..

The error message is missing margins. Or is it padding?

image

### PR DESCRIPTION
**Pull request by shahlin**
Signed-off-by: Shahlin Ibrahim <shahlin@28lightbulbs.com>

### Description

Please describe your pull request.

Fixes #16201

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: js/table/change.js [hunk 1]
```diff
@@ -155,6 +155,9 @@ function verifyAfterSearchFieldChange (index) {
     var $thisInput = $('input[name=\'criteriaValues[' + index + ']\']');
     // validation for integer type
     if ($thisInput.data('type') === 'INT') {
+        // Trim spaces if it's an integer
+        $thisInput.val($.trim($thisInput.val()));
```
Comment (williamdes) [inline js/table/change.js:159]:
```suggestion
        $thisInput.val(trim($thisInput.val()));
```
https://caniuse.com/#feat=mdn-javascript_builtins_string_trim

Comment (shahlin) [reply js/table/change.js]: Using the following:

```
$thisInput.val(($thisInput.val()).trim());
```

Comment (williamdes) [reply js/table/change.js]: Indeed, thank you ^^
Too much php code in my head ^^

**Comment by williamdes** [comment]
Thank you for this first contribution @shahlin !
I think you will find some helpful informations in #15942 about branch management

I squashed your work before merging :)

**Comment by shahlin** [comment]
> Thank you for this first contribution @shahlin !
> I think you will find some helpful informations in #15942 about branch management
> 
> I squashed your work before merging :)

Thanks @williamdes, glad to see the PR being merged. Will look into [#15942](https://github.com/phpmyadmin/phpmyadmin/pull/15942)! Thanks again

**Pull request merged**